themes/clouds_theme.lua

```diff
@@ -9,5 +9,6 @@ laptop.register_theme("Clouds", {
 	exit_textcolor = "#000000",
 	app_button = "blank.png",
 	textcolor = "#000000",
-	app_textcolor = "#0d7173",
+	app_textcolor = "#ffffff",
```

Comment:
app_textcolor needs to be "desktop_icon_label_textcolor" as proposed above, if you change to the new prefix